lib/opensusebasetest.pm

```diff
@@ -293,7 +293,8 @@ sub wait_boot {
         }
     }
 
-    unlock_if_encrypted;
+    # on svirt is system unlocked with workaround_type_encrypted_passphrase before this wait_boot
+    unlock_if_encrypted if !check_var('BACKEND', 'svirt');
```

Comment:
As suggested, please us `!check_var('BACKEND', 'svirt') and check_var('ARCH', 's390x')` or use `S390_ZKVM` variable